src/FamilyHubs.Referral.Web/Pages/ProfessionalReferral/Safeguarding.cshtml.cs

```diff
@@ -5,6 +5,8 @@ namespace FamilyHubs.Referral.Web.Pages.ProfessionalReferral;
 
 public class SafeguardingModel : PageModel
 {
+    public string BackUrl { get; set; } = default!;
```

Comment:
Could we rename this?
`BackUrl` made me think it was the Url to go back, but it looks like it's the Url to go onto the next page.

Reply:
Removed